Sources/Logging/Logging.swift

```diff
@@ -1,128 +1,40 @@
-        if self.logLevel <= level {
-            self.handler.log(level: level, message: message(), metadata: metadata(), error: error, file: file, function: function, line: line)
-        }
-    }
-
-    @inlinable
-    public func trace(_ message: @autoclosure () -> String, metadata: @autoclosure () -> Logging.Metadata? = nil, error: Error? = nil, file: StaticString = #file, function: StaticString = #function, line: UInt = #line) {
-        self.log(level: .trace, message: message, metadata: metadata, error: error, file: file, function: function, line: line)
-    }
-
-    @inlinable
-    public func debug(_ message: @autoclosure () -> String, metadata: @autoclosure () -> Logging.Metadata? = nil, error: Error? = nil, file: StaticString = #file, function: StaticString = #function, line: UInt = #line) {
-        self.log(level: .debug, message: message, metadata: metadata, error: error, file: file, function: function, line: line)
-    }
-
-    @inlinable
-    public func info(_ message: @autoclosure () -> String, metadata: @autoclosure () -> Logging.Metadata? = nil, error: Error? = nil, file: StaticString = #file, function: StaticString = #function, line: UInt = #line) {
-        self.log(level: .info, message: message, metadata: metadata, error: error, file: file, function: function, line: line)
-    }
-
-    @inlinable
-    public func warning(_ message: @autoclosure () -> String, metadata: @autoclosure () -> Logging.Metadata? = nil, error: Error? = nil, file: StaticString = #file, function: StaticString = #function, line: UInt = #line) {
-        self.log(level: .warning, message: message, metadata: metadata, error: error, file: file, function: function, line: line)
-    }
-
-    @inlinable
-    public func error(_ message: @autoclosure () -> String, metadata: @autoclosure () -> Logging.Metadata? = nil, error: Error? = nil, file: StaticString = #file, function: StaticString = #function, line: UInt = #line) {
-        self.log(level: .error, message: message, metadata: metadata, error: error, file: file, function: function, line: line)
-    }
-
-    @inlinable
-    public subscript(metadataKey metadataKey: String) -> Logging.Metadata.Value? {
-        get {
-            return self.handler[metadataKey: metadataKey]
-        }
-        set {
-            self.handler[metadataKey: metadataKey] = newValue
-        }
-    }
-
-    @inlinable
-    public var metadata: Logging.Metadata {
-        get {
-            return self.handler.metadata
-        }
-        set {
-            self.handler.metadata = newValue
-        }
-    }
-
-    @inlinable
-    public var logLevel: Logging.Level {
-        get {
-            return self.handler.logLevel
-        }
-        set {
-            self.handler.logLevel = newValue
-        }
-    }
-}
+//===----------------------------------------------------------------------===//
```

Comment:
@ianpartridge this adds an empty file with just a license header

Reply:
I think `public enum Logging` is still in here?